compliance_checker/cf/cf.py

```diff
@@ -491,11 +491,15 @@ def check_units(self, ds):
             # skip string type vars
             if v.dtype.char == 'S':
                 continue
-
+            
+            # skip quality control vars
+            if hasattr(v, 'flag_meanings'):
+                continue
+            
             units = getattr(v, 'units', None)
 
             # 1) "units" attribute must be present
-            presence = Result(BaseCheck.HIGH, units is not None, ('units', k, 'present'))
+            presence = Result(BaseCheck.MEDIUM, units is not None, ('units', k, 'present'))
```

Comment:
@daf I guess that's reasonable. Reliably identifying variables which legitimately don't have units because they're dimensionless will be tricky... but maybe can be dealt with separately.